src/goto-instrument/code_contracts.cpp

```diff
@@ -389,8 +388,8 @@ void code_contractst::operator()()
     goto_functions.function_map.find(CPROVER_PREFIX "initialize");
   assert(i_it!=goto_functions.function_map.end());
 
-  for(id_sett::const_iterator it=summarized.begin();
-      it!=summarized.end();
+  for(std::unordered_set<irep_idt>::const_iterator it = summarized.begin();
```

Comment:
Possibly in a separate commit, but this should likely be changed to `for(const auto &contract : summarized)`